modules/delegates.js

```diff
@@ -17,6 +17,7 @@
 const crypto = require('crypto');
 const _ = require('lodash');
 const async = require('async');
+const lisk = require('lisk-elements').default;
```

Comment:
I think for 1.0, we decided not to use `lisk-elements` in core, and when we are going to migrate, we should probably do it in other PR.

Reply:
Can anyone explain the reason to write duplicate code in 1.0 and where did that decision made? 

Don't we have trust on the `lisk-elements`?

Reply:
We do but it wasn't planned for now. We will make a full migration of our cryptography towards `lisk-elements` in consequent releases.

Reply:
@diego-G Really? What about performance?

Reply:
It is not the time to discuss that @4miners. When the moment arrives, we will measure if the performance affects us for real.

Reply:
I'm just worried, because you wrote `We will make a full migration of our cryptography towards lisk-elements in consequent releases.` - that means the decision was already made and I don't even know about it.

Reply:
Using `lisk-elements` in Core not only in the test suites but as a regular dependency, it is not new at all. We are talking about this since ages... Obviously, security it is the main expertise of `lisk-elements`. Are you sure you weren't aware? 
Of course, there is nothing 100% settled until we create issues an assign them to milestone. There is room for discussions if you have any concern.